<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>solver</class>
 <widget class="QMainWindow" name="solver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>solver</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="soup">
     <item row="0" column="0">
      <widget class="QWidget" name="widget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionLoad_words"/>
    <addaction name="actionLoad_Soup"/>
    <addaction name="actionSolve"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>44</height>
    </size>
   </property>
   <property name="floating">
    <bool>true</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>0</y>
       <width>191</width>
       <height>441</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
      <item>
       <widget class="QListWidget" name="FoundedList"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <action name="actionLoad_words">
   <property name="text">
    <string>Load words</string>
   </property>
  </action>
  <action name="actionLoad_Soup">
   <property name="text">
    <string>Load Soup</string>
   </property>
  </action>
  <action name="actionSolve">
   <property name="text">
    <string>solve</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
